desktop/WalletManager.ts

```diff
@@ -165,6 +162,15 @@ class WalletManager {
         port,
         meta: { salt: encryptionConst.DEFAULT_SALT }
       };
+      await this.nodeManager.activateNodeProcess();
```

Comment:
Pretty similar to a code on line 212, I'd prefer to extract it into another method and call it from these places.